scripts/EN/Harry

```diff
@@ -2732,22 +2732,22 @@ CleanScreen(); // 0x336e: 00000064
 LoadBackground(0x41); // 0x3372: 00001069
 DrawBackground(); // 0x3378: 00000065
 PlayMusic(0x6f, 0x1); // 0x337c: 0000206c
-ShowText((4001)"Without knowing why, I dialed my home number, where no one should be. Maybe there is a chance that Cheryl has already returned home... With slight hope I..."); // 0x3384: 0000100c
+ShowText((4001)"Without knowing why, I dialed my home number, W1where no one should be.¥nMaybe there is a chance that¥nCheryl has already returned home…¥nW2With slight hope I…"); // 0x3384: 0000100c
 HandleInput(); // 0x338a: 00000010
 PlaySoundFx(0x1d); // 0x338e: 00001072
-ShowText((4002)"(Bururururu...Bururururu...) (Onomatopoeia for ringing telephone) The phone is ringing... The mechanical electrical sound repeats over and over. Then all of a sudden it stopped."); // 0x3394: 0000100c
+ShowText((4002)"(Brrrrrrrrt…W4Brrrrrrrrt…)¥nW4The phone is ringing…¥nW2The mechanical electrical sound repeats over and over.¥nW2Then all of a sudden it stopped."); // 0x3394: 0000100c
```

Comment:
Changed to the English sound for how a phone rings when you are the one calling.